### DISCUSSION TITLE
ONVIF in 0.13?

### DISCUSSION THREAD

**Discussion by rykr** [blakeblackshear/frigate#8501] [Ask A Question] [answered]
I don't see anything in the 0.13 docs.  What is the new ONVIF support about?  how do I use it?

**Comment by NickM-27** [comment, marked as answer]
Where are you looking in the 0.13 docs? there are multiple pages

[Camera PTZ Autotracking](https://deploy-preview-6262--frigate-docs.netlify.app/configuration/autotracking)
[Camera PTZ controls and onvif config](https://deploy-preview-6262--frigate-docs.netlify.app/configuration/cameras#setting-up-camera-ptz-controls)

**Comment by rykr** [comment]
ok.  I went to https://deploy-preview-6262--frigate-docs.netlify.app and did a search for ONVIF and got nothing.  so this is just for PTZ cameras?  I thought it might have something to do with ONVIF events from the camera.

Thanks!

**Reply by NickM-27** [reply]
search doesn't work in the 0.13 docs. And yes onvif is for ptz control
